src/auth.js

```diff
@@ -94,6 +95,20 @@ class Auth {
     } catch (e) {
       this.core.logger.warn(e);
     }
+
+    /**
+     * @type {String}
+     * @private
+     * @readonly
+     */
+    this.refreshTokenSecret = core.config('session.options.refreshTokenSecret');
```

Comment:
If you put your session configuration options into something like `session.jwt`, then you can just do `this.options = core.config()`.

Reply:
But maybe it would be better to pass the options as an argument in the constructor ?